protocol/04_detection.md

```diff
@@ -0,0 +1,55 @@
+# Automated detection protocol
+
+#### These are instructions for running batdetect2 using the bat-detector-msds pipeline on recovered nighttime recordings
+
+## Pre-steps:
+
+- Fork `aditya-uw/bat-detector-msds` to access the batdetect2-pipeline branch in that repository.
+  - This is where the bash script that runs the pipeline is stored.
+- Download the forked repository with `git clone (your forked repository)` on the Linux machine with the mounted UBNA hard drives.
+- Follow the forked repository's README.md instructions about `pip install -r requirements.txt` and updating the submodule.
+  - It is ideal to install requirements and dependencies in a virtual environment with python=3.x.
```

Comment:
Maybe somewhere in here, you can write something about fetching and pulling changes you made in your own repository. This would be especially helpful in the case where someone else wants the bat call CSVs for certain deployments and you've already run the pipeline for them and updated your branch.

Also could you add something about switching to the batdetect2-pipeline?